src/main/java/ru/naumen/servacc/ui/UIController.java

```diff
@@ -318,7 +306,6 @@ public void widgetSelected(SelectionEvent e)
         // Filter events
         filteredTree.getFilter().addModifyListener(new ModifyListener()
         {
-            @Override
             public void modifyText(ModifyEvent e)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1161)